pkgs/development/python-modules/phik/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  nbconvert,
+  pytest-pylint,
+  numba,
+  pandas,
+  matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "wheel";
+    python = "py3";
+    abi = "none";
+    platform = "any";
+    sha256 = "c398452c5c1eea153905666b289c6a153712cf3d58811fa41e2bbbd27a65d678";
+  };
+
+  propagatedBuildInputs = [
+    nbconvert
+    pytest-pylint
+    numba
+    pandas
+    matplotlib
+  ];
+
+  # specify unpackPhase here to fix error when trying to unpack wheel file
+  unpackPhase = ''
+    mkdir -p dist
+    cp "$src" "dist/$(stripHash "$src")"
+  '';
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  meta = with lib; {
+    description = "Phi_K is a new and practical correlation coefficient based on several refinements to Pearson’s hypothesis test of independence of two variables.";
```

Comment:
This is quite a long description so should probably be the `longDescription`, and the `description` can just be "Phi_K correlation analyzer library"